resources/definitions/creality_sermoonv1.def.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "Creality Sermoon V1/V1 Pro",
+    "version": 2,
+    "inherits": "creality_base",
+    "metadata": {
+        "author": "Luis Gabriel Gomez",
+        "quality_definition": "creality_base",
+        "visible": true,
+        "has_variants": true
+    },
+    "overrides": {
+        "machine_name": {
+            "default_value": "Creality Sermoon V1"
+        },
+        "machine_width": {
+            "default_value": 175
+        },
+        "machine_depth": {
+            "default_value": 175
+        },
+        "machine_height": {
+            "default_value": 165
+        },
+        "machine_head_with_fans_polygon": {
+            "default_value": [
+                [-26, 34],
+                [-26, -32],
+                [32, -32],
+                [32, 34]
+            ]
+        },
+        "preferred_variant_name": "0.4mm Nozzle",
```

Comment:
I think the linter fails on this. Check if this line is correct if it is an override then it should be a dict I believe.

but it is kinda hard for me to double check this since I'm typing this from my phone atm.

Reply:
Will check. Linter isn't failing on this one, return is `Diagnostics: []`

Reply:
search the fdmprinter.def.json for `preferred_variant_name` results in nothing found.

I think you need to remove it.

Reply:
looking at the error in the workflow https://github.com/Ultimaker/Cura/actions/runs/3587770634/jobs/6038498603#step:8:30

tells me it gets a str were it expects a dict like all the other overrides

Reply:
the key, value should be defined in the metadata
https://github.com/Ultimaker/Cura/blob/eb4451f25cc029763b2c8d7529e7281c36d99108/resources/definitions/creality_base.def.json#L21

Reply:
but since you're setting it with the same value as the one defined in creality_base from which you inherit, you can omit it.